webapp/templates/transport/schedule.html

```diff
@@ -0,0 +1,48 @@
+{% extends 'base.html' %}
+{% load bootstrap3 %}
+{% block title %}Transport rooster{% endblock %}
+{% block content %}
+    <h1 class="page-header">Transport rooster</h1>
+
+    <!-- Table -->
+    <table class="table table-hover" id="schedule-table">
+        <thead>
+        <tr>
+            <th>Datum</th>
+            <th>Route</th>
+            <th>Rijder</th>
+            <th>Bijrijder</th>
+        </tr>
+        </thead>
+        <tbody>
+        {% for ride in object_list %}
+        <tr class="ride">
+            <td class="col-md-2"><a href="{% url 'ride' ride.slug %}">{{ ride.date }}</a></td>
```

Comment:
Can we use `date_str` here too? (or use the date formatter: https://docs.djangoproject.com/en/2.1/ref/templates/builtins/#date)

Reply:
I've improved the date formatting. I noticed that when I formatted in the model it seemed to follow a different locale (showing "March") than when I used this date formatted in the template (showing "Maart"). So I was kind of forced to use the template date formatter, but this meant repeating the format a lot. Please let me know if there is a better way.

Reply:
Honestly I hate that Django does that. I think repeating the format in the template is fine. Looks good this way.